### DISCUSSION TITLE
Issue with Azure Container Apps. CONTAINER_APP_REPLICA_NAME does not exist

### DISCUSSION THREAD

**Discussion by tbuha** [AsynkronIT/protoactor-dotnet#2122] [Ideas]
Hi there,
I tried to use it. And have an error in [line 24](https://github.com/asynkron/protoactor-dotnet/blob/dev/src/Proto.Cluster.AzureContainerApps/Services/EnvironmentContainerAppMetadataAccessor.cs#L24):
`
var replicaName = Environment.GetEnvironmentVariable("CONTAINER_APP_REPLICA_NAME") ?? throw new Exception("No replica name provided");
`

Have you ever tried to run it with Azure Container Apps?

**Comment by tbuha** [comment]
I looked through the code replicaName does not have any references but makes a problem :)
So I've done my own implementation:

`
            var replicaName = Environment.GetEnvironmentVariable("CONTAINER_APP_REPLICA_NAME") ?? revisionName;
`

**Comment by npehrsson** [comment]
RevisionName would not be unique per replica.

Environment.GetEnvironmentVariable("CONTAINER_APP_REPLICA_NAME") works for us in our environment.
Which Environment Type are you using?

**Reply by tbuha** [reply]
I use it for Azure Container App .NET 8 application. 
I tested CONTAINER_APP_REPLICA_NAME does not exist. 
Also I looked through the code and cant see any references on replicaName in the project Proto.Cluster.AzureContainerApps
